From bfd5509dc249af2f3363b05d5298953c6cf87bd2 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Wed, 22 Jul 2020 18:45:52 +0800
Subject: [PATCH] ARM: dts: rv11xx-evb-v10.dtsi update touch ic slave address

There is a  pull-up resistor added to the new touch board,
so the address of TP is changed to 0x14.
At the same time, switch the reporting coordinate method for
meet the requirement of QT using linuxfb.

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: I1e25f7e47b2b93ccb66b1c093241eb94afc1a574
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 14 +++-----------
 1 file changed, 3 insertions(+), 11 deletions(-)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index 3d4f849fd048..aa4d90cd2b06 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -929,11 +929,10 @@
 	status = "okay";
 	clock-frequency = <400000>;
 
-	gt1x: gt1x@5d {
+	gt1x: gt1x@14 {
 		compatible = "goodix,gt1x";
-		reg = <0x5d>;
-		pinctrl-names = "default";
-		pinctrl-0 = <&tp_int_pin>;
+		reg = <0x14>;
+		gtp_ics_slot_report;
 		power-supply = <&vcc18_lcd_n>;
 		goodix,rst-gpio = <&gpio2 RK_PB0 GPIO_ACTIVE_HIGH>;
 		goodix,irq-gpio = <&gpio2 RK_PB2 GPIO_ACTIVE_LOW>;
@@ -1082,13 +1081,6 @@
 			rockchip,pins = <0 RK_PB0 0 &pcfg_pull_up>;
 		};
 	};
-
-	gt1x {
-		/omit-if-no-ref/
-		tp_int_pin: tp-int-pin {
-			rockchip,pins = <2 RK_PB2 RK_FUNC_GPIO &pcfg_pull_up>;
-		};
-	};
 };
 
 &pmu_io_domains {
-- 
2.35.3

